index.html

```diff
@@ -148,6 +148,10 @@ <h1 id="aquaframe">The Magnificent Miss F.</h1>
       <article id="about">
         <h2>About</h2>
       </article>
+      <article id="home">
+        <h2>Welcome</h2>
+        <p>homepage content can go here</p>
+      </article>
```

Comment:
@MagnificentF don't add content here, cause this is the compiled html not the src/main.jade